### DISCUSSION TITLE
[공지사항] ※2021-01-09 수업 전 공지사항※

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#79] [00_공지사항]
여러분 안녕하세요

멘토 유혁준입니다.

여러분 수업 전 공지사항이 있어서 수업 전까지 꼭 해주셔야합니다.

1. lecture 파일에서 02 directory 들어가시면 [class 교육자료](https://github.com/4Moyede/HexatonClass-01/blob/main/lecture/02/20210109-AC_class_1_OOP.ipynb) 가 있습니다. 한 번 정독하시고 수업 들어오시면 됩니다.
2. 여러분들의 과제를 리뷰하였습니다.
   1. 여러분들의 과제 수정사항들을 확인해보시고 수업 전까지 완료하셔서 push하시고 pull request 다시 보내시길 바랄게요.
   2. 다른 멘티들이 올린 과제를 직접 다 들어가서 code를 보시고 여러분들이 code review를 각 멘티들마다 달아주시길 바랍니다. 리뷰 방법은 1,2,3 번 문제에 대한 code에 대해서 1개이상씩 정리하셔서 [1주차 과제 리뷰](https://github.com/4Moyede/HexatonClass-01/discussions/78) 밑에 댓글로 꼭 달아주시길 바랍니다.  양식은 자유이며, 저희가 올렸던 방식대로 코드리뷰 하셔도 좋습니다.
   3. code review는 "이런 부분이 좋았어요!", "이 부분은 이렇게 바꾸는건 어떨까요?" 등등 협업자에게 code를 제안을 하기도 합니다. 여러분들이 가지고 있던 생각들을 서로 공유하고, 토론하시길 바랄게요 :)
3. 1/9일 수업 때 늦지마시고 (늦으면 노래시킬거임) 15분 정도 일찍 줌 켜놓을거니까 오셔서 궁금한 사항 있으면 편하게 질문하셔도 좋습니다.

그럼 마지막 1주차 과제 끝까지 마무리 잘 하시고 궁금한 점 있으면 언제든지 이슈나 discussion에 올려주세요!

감사합니다.

**Comment by dev-LeoKang** [comment]
확인했습니다!

**Comment by lsj0822** [comment]
확인했습니다

**Comment by dlrudgnsdlek** [comment]
확인했습니다

**Comment by jiwoo4424** [comment]
확인했습니다!

**Comment by NA-DK** [comment]
확인 했습니다~
